### DISCUSSION TITLE
How do i set a Keystroke as an action?

### DISCUSSION THREAD

**Discussion by Michachacha** [StreamController/StreamController#119] [Q&A] [answered]
Can i execute a keystroke (like control c) when a button is pressed?

**Comment by Core447** [comment, marked as answer]
This can be done using the Hotkey action from the OS plugin. If you don't see said action please add [this](https://github.com/streamcontroller/osplugin?tab=readme-ov-file#hotkeys--write-text) udev rule to your system and add your user to the `input` group.

**Reply by Michachacha** [reply]
Thank you
